tests/foreman/api/test_multiple_paths.py

```diff
@@ -236,7 +236,10 @@ def test_post_unauthorized(self):
         @Assert: HTTP 401 is returned
 
         """
-        for entity_cls in valid_entities():
+        post_entities = list(valid_entities())
+        # TemplateKind is a valid entity but it is read-only.
+        post_entities.remove(entities.TemplateKind)
+        for entity_cls in post_entities:
```

Comment:
Can we use the same approach as in https://github.com/SatelliteQE/robottelo/pull/3029/files#diff-4d5fa73ca5b6eca8886d32b3df8fc316R265?  This will ease adding/removing entities with much code logic change.